packages/okta-angular/src/okta/okta.guard.ts

```diff
@@ -52,7 +52,12 @@ export class OktaAuthGuard implements CanActivate {
     /** 
      * Store the current path
      */
-    this.oktaAuth.setFromUri(state.url);
+    let path: Array<string> = [];
+    for (var i in route.url) {
+      path.push(route.url[i].path);
+    }
+
+    this.oktaAuth.setFromUri(path.join('/'), route.queryParams);
```

Comment:
Thought I would leave this here, but here is another way you could do this (trying to be helpful any way I can):
```javascript
const path: string = Object.keys(route.url)
  .map((val: UrlSegment) => val.path)
  .join('/');

this.oktaAuth.setFromUri(path, route.queryParams);
```